<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutEpargne</class>
 <widget class="QDialog" name="AjoutEpargne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AjoutEpargne</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>No de Compte :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Taux d'Interet :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Solde :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Description :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Compte">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Description">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Solde">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_TauxInteret">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAjouterEpargne">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>90</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAjouterEpargne</sender>
   <signal>clicked()</signal>
   <receiver>AjoutEpargne</receiver>
   <slot>slotConfirmerEpargne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotConfirmerEpargne()</slot>
 </slots>
</ui>
